nims-notes-things/src/components/notes.js

```diff
@@ -9,13 +10,17 @@ function Notes() {
 
   return (
     <div>
-      {notes.map((n) => {
+      {notes.map((note, index) => {
         return (
           <div>
             <Note>
-              <p>{n.name}</p>
-              <p>{n.date}</p>
-              <p>{n.note}</p>
+              <PinImage src={Pin} alt=""></PinImage>
```

Comment:
add alt text... `alt="pin-image"` for example

Reply:
really just so that if the image doesn't display for some odd reason something will still display to the user